docs/content/policy-reference.md

```diff
@@ -450,8 +450,10 @@ The following table shows examples of how ``glob.match`` works:
 | <span class="opa-keep-it-together">``output := urlquery.decode_object(string)``</span> | ``output`` is ``object`` deserialized from a URL query parameter string |
 | <span class="opa-keep-it-together">``output := json.marshal(x)``</span> | ``output`` is ``x`` serialized to a JSON string |
 | <span class="opa-keep-it-together">``output := json.unmarshal(string)``</span> | ``output`` is ``string`` deserialized to a term from a JSON encoded string |
+| <span class="opa-keep-it-together">``output := json.is_valid(string)``</span> | ``output`` is a ``boolean`` that indicated whether ``string`` is a valid JSON document |
 | <span class="opa-keep-it-together">``output := yaml.marshal(x)``</span> | ``output`` is ``x`` serialized to a YAML string |
 | <span class="opa-keep-it-together">``output := yaml.unmarshal(string)``</span> | ``output`` is ``string`` deserialized to a term from YAML encoded string |
+| <span class="opa-keep-it-together">``output := yaml.is_valid(string)``</span> | ``output`` is a ``boolean`` that indicated whether ``string`` is a valid YAML document that can be decoded by `yaml.unmarshal` |
```

Comment:
Nit: We should use a similar format for both descriptions. For example `output is true if string is a valid JSON document` and `output is true if string is a valid YAML document`.

Reply:
The problem is that the yaml one is different. It is verifying that it can be unmarshalled with `yaml.unmarshal`, which supports a subset of yaml, (eg a "valid" yaml string passed in can give back false if it cannot be unmarshalled) whereas the json check is able to confidently determine if something is valid or not.

Reply:
🤔 - then again with https://github.com/open-policy-agent/opa/pull/2717 merged its potentially kind of a moot-point from a use-case perspective now

Reply:
Yeah, I agree it's going to be less useful but at least in our case we'll still be running with `strict-builtin-errors` for now.